src/components/CustomerQuota/ItemsSelection/ItemsSelectionCard.tsx

```diff
@@ -21,6 +21,7 @@ import {
 } from "../../../context/alert";
 import { validateAndCleanId } from "../../../utils/validateIdentification";
 import { CampaignConfigContext } from "../../../context/campaignConfig";
+import i18n from "i18n-js";
 
 interface ItemsSelectionCard {
```

Comment:
In `ItemNoQuota.tsx` file, we missed out "Cannot purchase"/"Cannot redeem" label